server/src/infra/graphql/mutation.rs

```diff
@@ -163,6 +163,48 @@ fn unpack_attributes(
     })
 }
 
+fn consolidate_attributes(
```

Comment:
Can you add a comment describing the purpose of the function? It's clear in the PR context, but not necessarily in general. And maybe pull up the comment about the relative priority, both here and in the GraphQL schema (by adding it to the correct struct)

Reply:
Added a comment, and to the fields in the GraphQL schema.